playbook-servers-ioc-docker-epics.yml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: docker_image
+  remote_user: sirius
+  become: true
+
+  pre_tasks:
+    - name: Include distribution-dependent variables
+      include_vars: "{{ item }}"
+      vars:
+        possible_var_files:
+          - "group_vars/{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml"
+          - "group_vars/{{ ansible_distribution }}.yml"
+          - "group_vars/{{ ansible_os_family }}.yml"
+      loop: "{{ q('first_found', possible_var_files, errors='ignore') }}"
+
+  roles:
+    - role: lnls-ans-role-repositories
+    - role: lnls-ans-role-users
+      when: global_repositories_role | default(true) | bool
+    # - role: lnls-ans-role-ntp
+    - role: lnls-ans-role-python
```

Comment:
We should keep this up in our radar.
A cleaner approach is to use official supported images such as [the python images from dockerhub](https://hub.docker.com/_/python?tab=tags&page=1&ordering=last_updated&name=stretch).
For an initial version It will work.

Reply:
yes yes. a LOT can be simplified and optimized now that IOCs are run in containers. We will do this gradually with time. 
At this time, as I see, the most important thing is to have an initial system config for testing as soon as possible how stable the IOCs wil run as docker services in our servers

Reply:
@carneirofc , I started testing python:3.6.13-slim-stretch. adding a few symlinks and additional python packages we need to add when building the image, we will be able to use this image pretty soon, thus avoiding the complex ansible python role and creating slimmer images.

ps: do you know the diff between python:3.6.13-slim-stretch and python:3.6.13-stretch ?

Reply:
The slim image will contain the bare minimum to run python and various unix programs will not be installed.
The main goal is to make the image as small as possible.
The default  image would work just fine, this is a "minor" optimization.

https://github.com/docker-library/python/blob/master/3.7/stretch/Dockerfile
https://github.com/docker-library/python/blob/master/3.7/stretch/slim/Dockerfile

Reply:
In retrospect, I think we should start by using the normal non-slim version of the Debian image; or at least give a special attention when testing. Due to the lack of commonly used programs, simple things may break out of nowhere.
An example that happened last week, is that I was having problems when cloning git repositories because the package that handles CA certificates is not present by default ...